support/get-logs.sh

```diff
@@ -10,54 +10,123 @@
 # disclosure restricted by GSA ADP Schedule Contract with IBM Corp.
 #
 
-PROGRAM_NAME=$0
-VERSION="2020.1.3"
+PROGRAM_NAME="${0}"
+VERSION="2020.1.4"
 DATE=`date +%d-%m-%y`
 TIME=`date +%H-%M-%S`
 
+####################################################################################################
+# Script helper functions
+####################################################################################################
+
 usage() {
-    printf "$PROGRAM_NAME v${VERSION}\n\n"
+    printf "${PROGRAM_NAME} v${VERSION}\n\n"
```

Comment:
Unnecessary use of curly braces. Only recommended when the variable is touching another character like with `v` in v${VERSION}

Reply:
only recommended then really?
I thought they were always recommended as it doesn't hurt

Reply:
It won't hurt I guess, I only had them where required before